src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -877,14 +958,31 @@
         this.approvalRequestDAO.put(rejectedApprovalRequest).then(o => {
           this.approvalRequestDAO.cmd(this.AbstractDAO.RESET_CMD);
           this.finished.pub();
-          this.notify(this.SUCCESS_REJECTED, '', this.LogLevel.INFO, true);
+          this.notify(this.SUCCESS_ASSIGNED, '', this.LogLevel.INFO, true);
 
           X.stack.back();
         }, e => {
           this.throwError.pub(e);
           this.notify(e.message, '', this.LogLevel.ERROR, true);
         });
       }
+    },
+    {
+      name: 'assignRequestL',
```

Comment:
the trailing L - intentional?

Reply:
Follow the other listeners, assumed the L was for Listener

Reply:
I'm not aware of that convention. Which ones do you have in mind?

Reply:
approveWithMemoL, was added from this syntax PR: https://github.com/foam-framework/foam2/pull/4223

So read that it was because there was an approveWithMemo action of the same name. So I can remove the trailing L from this one.